sag_et.par

```diff
@@ -15,88 +19,126 @@ n 14 8
 n 15 9
 /|  N\  /N\
 n 16 9
-)N(  M)
+|)  (|\
 d 17 10
 /|\  /N\
 d 19 11
 /N\
+n 20 12
+)~|  (|\  ~N\  /N\
 n 21 12
-|)  |\
+|)  (|\
 d 22 13
 s 11
 /|  N\  /N\
 n 23 13
-)N(  M)  (X(
+|)  |\  (|\
 d 24 14
 s 8
 /|\  /N\
+n 25 15
+)~|  /|  N\  ~N\  /N\
 d 26 15
+s 13
 /N\
 d 27 16
-/|  /|)  N\  /N\
+/|  (|\  N\  /N\
+n 28 16
+|)  |\  (|\
 d 29 17
 /|  N\  /N\
+n 30 18
+)|  /|  (|\  N\  (N~  /N\
 d 31 18
 /|\  /N\
+n 32 19
+)~|  /|  N\  ~N\  /N\
+n 33 19
+|)  |\  (|\
 d 34 20
 /|  /|\  N\  /N\
+n 35 20
+|(  |)  (|\
 d 36 21
-s 9 18
+s 18
 |)  N)  /N\
+n 37 22
+)|  /|  (|\  N\  (N~  /N\
 d 38 22
 /|\  /N\
 d 39 23
-s 13

```

Comment:
Scala only supports one subset notation for a given EDO. I went with the periodic table's recommendation of 26 over the old recommendation of 39 from the Xenharmonikon article.

Reply:
That's good.